modules/mauth-common/src/main/java/com/mdsol/mauth/MAuthRequest.java

```diff
@@ -150,6 +195,14 @@ public MAuthVersion getMauthVersion() {
     return mauthVersion;
   }
 
+  public void setBodyInputStream(InputStream bodyInputStream) {
+    this.bodyInputStream = bodyInputStream;
+  }
```

Comment:
Please remove this as it is no longer used and is problematic because it bypasses our constructor validation

Reply:
removed https://github.com/mdsol/mauth-jvm-clients/pull/132/commits/ac9b72d9407033a365bbc428dfdd2315ead45826